x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/delete_button.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { ml } from '../../../../services/ml_api_service';
```

Comment:
please use 
```typescript
  const {
    services: {
      mlServices: { mlApiServices },
    },
  } = useMlKibana();
```
instead of a static import